### PR TITLE
Charles user

### PR DESCRIPTION
**Pull request by Charles-Hong520**
I created another branch for some reason, but this is the new and working one. I was able to link problem controller, service, and repository together. The dummy testing data is in resources/data.sql. It will be uploaded into the embedded h2 db upon start up, so it is not stored there forever. I did this just for quick testing, but we can use that for unit tests later. This way there won't be any hard coding in the problem service class. Every Problem class returns a proper function call and there are no more hard codes.

### CODE REVIEW

File: src/main/java/com/monkeyrizz/codeforest/model/Problem.java
Comment (ryanchangx) [inline src/main/java/com/monkeyrizz/codeforest/model/Problem.java:1]: use lombok to make your getters and setters

File: src/main/java/com/monkeyrizz/codeforest/model/Solution.java
Comment (ryanchangx) [inline src/main/java/com/monkeyrizz/codeforest/model/Solution.java:1]: ditto

File: src/main/java/com/monkeyrizz/codeforest/model/User.java
Comment (ryanchangx) [inline src/main/java/com/monkeyrizz/codeforest/model/User.java:1]: ditto

**Review by ryanchangx — COMMENTED** [review]
use lombok man

**Comment by Charles-Hong520** [comment]
added lombok in latest push

**Review by ryanchangx — APPROVED** [review]
LGTM

**Pull request merged**